mozilla/foundation.mozilla.org
# update copy pt one

Closes #7701


**Link to sample test page**: https://foundation-s-7701-updat-um8fbp.herokuapp.com/en/privacynotincluded/about/methodology/

**Steps to test**: 

1. Please visit the test link above and take a look at the new copy.
2. If everything looks as expected, testing is complete!
